src/types/flexible_gridgallery.ts

```diff
@@ -0,0 +1,30 @@
+import type { MediaItemType } from '@/types/types'
+
+export interface FlexibleGridGalleryItem {
+  id: string
+  sectionSummary: string
+  sectionTitle: string
+  subtitle: string
```

Comment:
I just looked at the graphQL query https://github.com/UCLALibrary/library-website-nuxt/blob/main/gql/fragments/BlockGridGalleryCardsFragment.gql, we can safely remove subtitle and summary here.